src/workloads/filesystem/Flushing.yml

```diff
@@ -0,0 +1,83 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-perf"
+Description: >
+  Run large insertMany's to stress checkpointing, journal flush and dirty memory with {w:majority}.
+  The workload runs a single CrudActor targeting:
+    - 'test' database, configurable via the Database parameter.
+    - 'Collection0' collection, configurable via the Collection parameter.
+    - 10 threads, configurable via the Threads parameter.
+    - 5000 iterations, configurable via the Iterations parameter.
+    - 1500 documents, configurable via the Documents parameter.
+    - {w:majority} insert options, configurable via the Options parameter.
+  The default documents have a very simple / flat structure as follows:
+    - _id: ObjectId
+    - x: a ~1KB fixed string 
+    - id: 'User%07d' incrementing field.
+  There are no additional indexes (just _id).
+  The important ftdc metrics to look at are:
+    - ss opcounters insert
+    - ss average latency writes
+    - system memory dirty
+    - ss wt log log sync time duration
+    - ss wt transaction transaction checkpoint currently running
+
+Keywords:
+  - Flushing
+  - IO
+  - bulk insert
+  - stress
+
+GlobalConstants:
+  - &database test
+  - &collection Collection0
+  - &maxPhases 1
+  - &insertThreads 10
+  - &iterationsCount 5000
+  - &defaultOptions
+    WriteConcern:
+      Level: majority
+  - &defaultDocument  # The size of each document is about 1kb.
+    x: {^PreprocessorFormatString: {format: "x%*s", withArgs: [1000, "x" ]}}
+    id: { ^Join: { array: [ { ^FormatString: { "format": "User%07d", "withArgs": [ { ^Inc: { start: 1 } } ] } } ] } }
+  - &documentList10x {^FlattenOnce: [
```

Comment:
This is both so awesome and so horrible at the same time!

Reply:
agreed, it is much better than the previous version but I would prefer:

```
  - &defaultDocument  # The size of each document is about 1kb.
    x: {^PreprocessorFormatString: {format: "x%*s", withArgs: [1000, "x" ]}}
    id: { ^Join: { array: [ { ^FormatString: { "format": "User%07d", "withArgs": [ { ^Inc: { start: 1 } } ] } } ] } }
  - &defaultDocumentList {^FlattenOnce: {of: *defaultDocument, number: 1500}}
```

> FWIW, we've had an extended slack conversation on this topic. I'll ping you on it.

In terms of this, I prefer this PreprocessorFormatString version given that it is now all static, it could be a bit clearer though. And it would be nice if it supported the python format function.